test/test-lib/resin-token.ts

```diff
@@ -0,0 +1,8 @@
+import * as resinToken from 'resin-token';
+import * as temp from 'temp';
+
+const trackedTemp = temp.track();
+
+export = resinToken({
```

Comment:
Can you please switch to `export const ... = resinToken('...')` format? And possibly also rename the file to balena-token, or just token instead?

Reply:
Sure, I can take a look at that